include/seqan3/alignment/scoring/detail/simd_matrix_scoring_scheme.hpp

```diff
@@ -28,9 +28,7 @@ namespace seqan3::detail
  * \tparam simd_score_t The type of the simd vector; must model seqan3::simd::simd_concept.
  * \tparam alphabet_t The type of the alphabet over which to define the scoring scheme; must model seqan3::semialphabet.
  * \tparam alignment_t The type of the alignment to compute; must be either seqan3::align_cfg::method_global or
- *                     seqan3::align_cfg::method_local.
- * \tparam scoring_scheme_t The type of the scoring scheme which will be used. Must model seqan3::scoring_scheme with
- *                          the given `alphabet_t`.
+ *                     seqan3::detail::method_local.
```

Comment:
is method local in `detail` :thinking: ?

Reply:
no, but it seems to be fixed in a later commit?